### PR TITLE
doc: matter: Added information about Amazon FFS support

### PR DESCRIPTION
**Pull request by kkasperczyk-no**
There is no clear statement in the documentation that Nordic
supports Amazon FFS in Matter and how to enable it.

Signed-off-by: Kamil Kasperczyk <kamil.kasperczyk@nordicsemi.no>

### CODE REVIEW

File: doc/nrf/links.txt [hunk 1]
```diff
@@ -520,6 +520,7 @@
 .. _`Alexa.Gadget.StateListener interface`: https://developer.amazon.com/en-US/docs/alexa/alexa-gadgets-toolkit/alexa-gadget-statelistener-interface.html
 .. _`Alexa Gadgets Notifications interface`: https://developer.amazon.com/en-US/docs/alexa/alexa-gadgets-toolkit/notifications-interface.html
 .. _`Alexa Gadgets Pairing`: https://www.amazon.com/gp/help/customer/display.html?nodeId=GV66YEZ2245F4QYS
+.. _`Amazon Frustration-Free Setup`: https://developer.amazon.com/docs/frustration-free-setup/matter-simple-setup-overview.html
```
Comment (LuDuda) [inline doc/nrf/links.txt:523]:
```suggestion
.. _`Amazon Frustration-Free Setup (FFS)`: https://developer.amazon.com/docs/frustration-free-setup/matter-simple-setup-overview.html
```

File: doc/nrf/ug_matter_configuring_protocol.rst [hunk 2]
```diff
@@ -87,12 +87,20 @@ These can be used for various purposes, such as dividing devices into groups (by
...
 
+* :kconfig:`CONFIG_CHIP_DEVICE_VENDOR_ID` sets the device manufacturer identifier that is assigned by the Connectivity Standards Alliance.
+* :kconfig:`CONFIG_CHIP_DEVICE_PRODUCT_ID` sets the product identifier that is assigned by the product manufacturer.
 * :kconfig:`CONFIG_CHIP_DEVICE_TYPE` sets the type of the device using the Matter Device Type Identifier, for example Door Lock (0x000A) or Dimmable Light Bulb (0x0101).
 * :kconfig:`CONFIG_CHIP_COMMISSIONABLE_DEVICE_TYPE` enables including an optional device type subtype in the commissionable node discovery record.
   This allows filtering of the discovery results to find the nodes that match the device type.
 * :kconfig:`CONFIG_CHIP_ROTATING_DEVICE_ID` enables an optional rotating device identifier feature that provides an additional unique identifier for each device.
   This identifier is similar to the serial number, but it additionally changes at predefined times to protect against long-term tracking of the device.
 
+The nRF Connect platform supports `Amazon Frustration-Free Setup`_ that allows to automatically setup Matter devices and join to Amazon Echo's network.
```
Comment (LuDuda) [inline doc/nrf/ug_matter_configuring_protocol.rst:98]:
```suggestion
The nRF Connect platform supports `Amazon Frustration-Free Setup`_ that allows Matter devices to be automatically commissioned to the Matter network, by help of the Matter-enabled Amazon Echo device.
```

File: doc/nrf/ug_matter_configuring_protocol.rst [hunk 3]
```diff
@@ -87,12 +87,20 @@ These can be used for various purposes, such as dividing devices into groups (by
...
   This allows filtering of the discovery results to find the nodes that match the device type.
 * :kconfig:`CONFIG_CHIP_ROTATING_DEVICE_ID` enables an optional rotating device identifier feature that provides an additional unique identifier for each device.
   This identifier is similar to the serial number, but it additionally changes at predefined times to protect against long-term tracking of the device.
 
+The nRF Connect platform supports `Amazon Frustration-Free Setup (FFS)`_ that allows Matter devices to be automatically commissioned to the Matter network, by help of the Matter-enabled Amazon Echo device.
+To enable support you need to met the setup prerequisites, by setting the following configuration options:
+* :kconfig:`CONFIG_CHIP_COMMISSIONABLE_DEVICE_TYPE` to ``y``
+* :kconfig:`CONFIG_CHIP_ROTATING_DEVICE_ID` to ``y``
+* :kconfig:`CONFIG_CHIP_DEVICE_TYPE` to the appropriate value dependent on the device used and compliant with the Matter Device Type Identifier.
+
```
Comment (greg-fer) [inline doc/nrf/ug_matter_configuring_protocol.rst:103]:
```suggestion

Amazon FFS support
==================

Matter in the |NCS| supports `Amazon Frustration-Free Setup (FFS)`_ that allows Matter devices to be automatically commissioned to the Matter network using the Matter-enabled Amazon Echo device.
To enable the FFS support, set the following configuration options to meet the Amazon FFS setup prerequisites:

* :kconfig:`CONFIG_CHIP_COMMISSIONABLE_DEVICE_TYPE` to ``y``.
* :kconfig:`CONFIG_CHIP_ROTATING_DEVICE_ID` to ``y``.
* :kconfig:`CONFIG_CHIP_DEVICE_TYPE` to the appropriate value, depending on the device used.
  The value must be compliant with the Matter Device Type Identifier.

```

**Review by greg-fer — CHANGES_REQUESTED** [review]

**Comment by greg-fer** [comment]
Not sure about you, but I would add an entry to changelog for this (Documentation section).

**Comment by kkasperczyk-no** [comment]
> Not sure about you, but I would add an entry to changelog for this (Documentation section).

@greg-fer ok, I did it.

**Review by LuDuda — APPROVED** [review]

**Review by Damian-Nordic — APPROVED** [review]

**Review by greg-fer — APPROVED** [review]

**Pull request merged**